### PR TITLE
arm64: dts: qcom: apq8016-samsung-matissevewifi: add accelerometer

### PR DESCRIPTION
**Pull request by rvlander**

### CODE REVIEW

File: arch/arm64/boot/dts/qcom/apq8016-samsung-matissevewifi.dts [hunk 1]
```diff
@@ -493,6 +514,18 @@
...
 
+	accel_int_default: accel_int_default {
+		pinmux {
+			function = "gpio";
+			pins = "gpio115";
+		};
+		pinconf {
+			pins = "gpio115";
+			drive-strength = <2>;
+			bias-pull-disable;
```
Comment (Minecrell) [inline arch/arm64/boot/dts/qcom/apq8016-samsung-matissevewifi.dts]:
```suggestion
			bias-disable;
```

File: arch/arm64/boot/dts/qcom/apq8016-samsung-matissevewifi.dts [hunk 2]
```diff
@@ -264,6 +264,27 @@
...
+	accelerometer@1d {
+		compatible = "st,lis2hh12";
+
+		vdd-supply = <&pm8916_l17>;
+		vddio-supply = <&pm8916_l5>;
+
+		reg = <0x1d>;
+		interrupt-parent = <&msmgpio>;
+		interrupts = <115 IRQ_TYPE_EDGE_RISING>;
+		interrupt-names = "INT1";
```
Comment (Minecrell) [inline arch/arm64/boot/dts/qcom/apq8016-samsung-matissevewifi.dts]: just curious, is this interrupt name used and/or suggested somewhere?

Comment (rvlander) [reply arch/arm64/boot/dts/qcom/apq8016-samsung-matissevewifi.dts]: I search to other examples of usage of the `lis2de12`, and they used it. I thought I'd put it here for documentation.

Comment (Minecrell) [reply arch/arm64/boot/dts/qcom/apq8016-samsung-matissevewifi.dts]: Makes sense, thanks!

**Pull request merged**